docs/appendices/staticAzure.rst

```diff
@@ -82,7 +82,7 @@ To use the template to set up a Toil Mesos cluster on Azure, follow these steps.
 |                             | mindful of your Azure subscription's core      |
 |                             | limits. Also be mindful of how many cores and  |
 |                             | how much disk and memory your Toil jobs will   |
-|                             | need: if any requirements is greater than that |
+|                             | need: if any requirements are greater than that |
```

Comment:
The `|` characters need to line up.  If you click on the `view` button, you'll notice the table for this code isn't showing up: https://github.com/DataBiosphere/toil/blob/e4652d2b7381bcbbc583eaf93d4babbdc283a3a6/docs/appendices/staticAzure.rst

It's a really weird convention in rst, but the `|` characters need to line up or it won't build the table.  Though great catch on the grammar!